search/src/main/resources/config/liquibase/changelog/20230124-add-search-list-column-table.xml

```diff
@@ -48,4 +48,9 @@
             <column name="default_sort" type="varchar(10)"/>
         </createTable>
     </changeSet>
+    <changeSet author="Ritense" id="add-unique-constraint-search_list_column">
```

Comment:
This changeset fails when running it

Reply:
I dont have any errors here. could you please show them to me?

Reply:
Seems like it was an issue in my DB